app/workers/education_form/process10203_submissions.rb

```diff
@@ -151,14 +151,17 @@ def repeat_form?(unprocessed_form, processed_form)
     #
     # This is only checking EVSS data until form questions that affect setting to DENIED have been reviewed
     def process_submission(submission, user_has_poa)
-      # This effectively disables the auto decision process
-      # Uncommenting the below logic and removing the following line will re-enable.
-      # status = if more_than_six_months?(submission.education_stem_automated_decision&.remaining_entitlement)
-      #            EducationStemAutomatedDecision::DENIED
-      #          else
-      #            EducationStemAutomatedDecision::PROCESSED
-      #          end
-      status = EducationStemAutomatedDecision::PROCESSED
+      # rubocop:disable Style/ConditionalAssignment
```

Comment:
Let's not ignore rubocop here. You can assign a conditional by adding a return after the `=`.